Mathlib/RingTheory/AdjoinRoot.lean

```diff
@@ -548,6 +548,15 @@ theorem minpoly_powerBasis_gen_of_monic (hf : f.Monic) (hf' : f ≠ 0 := hf.ne_z
     minpoly K (powerBasis hf').gen = f := by
   rw [minpoly_powerBasis_gen hf', hf.leadingCoeff, inv_one, C.map_one, mul_one]
 
+theorem _root_.finrank_quotient_span_eq_natDegree {f : K[X]} :
```

Comment:
Can you check this one is not already there? I have the impression to have seen it recently (but maybe it was in one of your PRs...).

Reply:
This version does not exists. One can prove a version over a ring that requires `f` to be a monic using `IsAdjoinRoot`. I added this version too and docstrings so that each version refers to the other.

Reply:
Maybe you were thinking of [finrank_quotient_span_eq_natDegree_norm](https://leanprover-community.github.io/mathlib4_docs/Mathlib/LinearAlgebra/FreeModule/Norm.html#finrank_quotient_span_eq_natDegree_norm)?